app/store/asyncStorage.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This file contains helper functions to store and retrieve key-values from async storage.
+ */
+
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+const storeData = async (key, value) => {
```

Comment:
This file is interesting and liked it, would like to understand the running scenarios.

Reply:
Yeah, lets discuss scenarios during our next sync-up